pkg/controller/pxc/tls.go

```diff
@@ -45,30 +45,27 @@ func (r *ReconcilePerconaXtraDBCluster) reconsileSSL(cr *api.PerconaXtraDBCluste
 }
```

Comment:
1) lines 19-21 -- we should generate certificates if `cr.Spec.TLS.IssuerConf != nil` even if `cr.Spec.AllowUnsafeConfig`

2) lines 36-45 -- we should not generate cert manually if `cr.Spec.TLS.IssuerConf != nil`